app/webpack/observations/uploader/components/time_shifter.jsx

```diff
@@ -141,6 +129,14 @@ class TimeShifter extends SelectionBasedComponent {
         <div className="tickmarks">
           {[-24, -12, 0, 12, 24].map( tick => <span className="tick" key={tick.toString( )}>{tick}</span> )}
         </div>
+        <div className="time-shifter-buttons">
+          <button className="btn btn-sm btn-primary" type="button" onClick={this.handleValueChange}>
+            {I18n.t( "shift" )}
+          </button>
+          <button className="btn btn-sm btn-default" type="button" onClick={this.resetShifter}>
+            {I18n.t( "cancel" )}
+          </button>
+        </div>
```

Comment:
Can we hide these when the offset is zero? Right now if it's zero, these buttons do nothing.